scripts/ingest_smoke_test.py

```diff
@@ -0,0 +1,232 @@
+"""Launch ingest pipeline smoke test
+
+DESCRIPTION
+Launch single study creation followed by file upload using manage-study
+OR concurrent ingest requests for X studies (stress test)
+
+PREREQUISITES
+set up access token on command line
+ACCESS_TOKEN=`gcloud auth print-access-token`
+
+EXAMPLE
+Creates N new study/ies in staging
+python ingest_smoke_test.py --token=$ACCESS_TOKEN -n <number of studies>
+
+Uses existing study, 'createTest' (SCP88 in staging), uploads 8K data set
+# Usage conditions: must have VPN access to staging server
+#                   files being uploaded cannot already exist in createTest
+python ingest_smoke_test.py --token=$ACCESS_TOKEN --dev-run
+"""
+
+import argparse
+import subprocess
+from datetime import datetime
+
+DATA = {
```

Comment:
While this is great for our purposes, I'm hesitant to hard code these files in a public repo, since I'm not sure all of this data is public.  I'm fairly certain the data from the staging studies is not (it may not have ever been published, so I'm not sure where that leaves us).

Reply:
As long as these buckets are private (and it seemed to require me to login to access them), I don't think using them for testing is materially different than having the data live on our staging servers.  However, I might suggest reaming the files as something generic like "scp-large-test-[x].tsv", to address the concern that we're expressing favortism to certain studies/groups in our source, and then putting a README in the google bucket that describes where the files come from

Reply:
Agreed. Files renamed `ef57bde`

Reply:
That would assuage my concerns